aws/aws-how-to/instances/launch-ubuntu-desktop.rst

```diff
@@ -1,78 +1,105 @@
-Launch an Ubuntu desktop on EC2
-===============================
+Launch and connect to an Ubuntu Mantic desktop EC2 instance
+===========================================================
 
-To create an Ubuntu desktop environment on your EC2 VM, you can use TightVNC as your remote desktop server and Remmina as your local desktop client.
+Use the xrdp server and Remmina to connect to an AWS EC2 instance of Ubuntu Mantis.
 
-.. Note::
+Select and configure Ubuntu Mantic 
+----------------------------------
 
-    Both the remote EC2 instance and your local machine are assumed to be running Ubuntu 16.04 or greater.
+Open AWS Marketplace and search for Ubuntu Mantic. Subscribe and agree to the Terms and Conditions. Select Launch Instance to configure.
 
+Select a region closest to you. For exapmple, if you're in the ``UK EU-West-2`` would be a good option. Click :guilabel:`Continue to Launch Through EC2 Instance`.
 
-Install Ubuntu desktop and TightVNC on your VM
-----------------------------------------------
+Give your instance a name. Mantic requires least two cores and at least 8gb RAM, which should be the default option. The default volume space of 8gb is the minimum required, however, more space will be required to install applications.
 
-To install Ubuntu desktop and TightVNC server on your EC2 instance, SSH into it and run:
+Create a security group
+-----------------------
+
+Select :guilabel:`Create Security Group`. Make sure the ssh tick box is selected to allow remote access.
```

Comment:
Use 'SSH' instead 'ssh' everywhere.
The post gives details about opening the RDP port, which is missing here.